extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -192,11 +179,23 @@ export class AmpAdApiHandler {
    * @private
    */
   updateSize_(height, width, source, origin) {
-    this.baseInstance_.attemptChangeSize(height, width).then(() => {
-      this.sendEmbedSizeResponse_(
-        true /* success */, width, height, source, origin);
-    }, () => this.sendEmbedSizeResponse_(
-        false /* success */, width, height, source, origin));
+    let newHeight, newWidth;
```

Comment:
1. `amp-iframe` checks for attribute 'resizable' while `amp-ad` not.
2. `amp-iframe` does not allow resize height to less than 100px.
3. 'amp-ad` sends back resize response back, while `amp-iframe` doesn't.
Were these difference intentional or were introduced because of different PRs.

You mean add docs comments to the code? will do that